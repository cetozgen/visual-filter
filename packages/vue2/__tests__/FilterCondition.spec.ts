const FilterCondition = require("../src/VueVisualFilter/FilterCondition.vue")

describe("FilterCondition.vue", () => {
  it("should pass", () => {
    // ..
  })
})
