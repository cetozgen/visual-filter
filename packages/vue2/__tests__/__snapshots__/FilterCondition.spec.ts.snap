// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`field updation logic should emit updateField on change event with correct paramaters 1`] = `
Array [
  Object {
    "argument": "John",
    "dataType": "nominal",
    "fieldName": "Last Name",
    "method": "contains",
    "type": "condition",
  },
  "Last Name",
]
`;

exports[`field updation logic should update modeled property 1`] = `"Last Name"`;

exports[`method updation logic should update modeled property 1`] = `"startsWith"`;
